airflow/providers/databricks/hooks/databricks_base.py

```diff
@@ -63,6 +64,8 @@
 DEFAULT_DATABRICKS_SCOPE = "2ff814a6-3304-4ab8-85cb-cd0e6f879c1d"
 OIDC_TOKEN_SERVICE_URL = "{}/oidc/v1/token"
 
+DEFAULT_AZURE_CREDENTIAL_SETTING_KEY = "use_default_azure_credential"
```

Comment:
This key should  also be added to the `extra_parameters` list in line 89 I think.